web/components/outcome-label.tsx

```diff
@@ -84,11 +84,12 @@ export function FreeResponseOutcomeLabel(props: {
   if (!chosen) return <AnswerNumberLabel number={resolution} />
   return (
     <FreeResponseAnswerToolTip text={chosen.text}>
-      <AnswerLabel
+      <span className="text-blue-400">TOP</span>
```

Comment:
Is this for testing? Did you mean to put back the commented out AnswerLabel?

Reply:
Might have to rearrange the card for resolved FR markets to show the resolved answer at the bottom of the card.

Reply:
Think we're already doing this: 
![image](https://user-images.githubusercontent.com/856034/169962458-16f711b2-ffde-4e3a-a28a-8f224ffd5df1.png)

Removing the commented code!